src/LimitStream.php

```diff
@@ -75,7 +75,7 @@ public function seek($offset, $whence = SEEK_SET)
     {
         if ($whence !== SEEK_SET || $offset < 0) {
             throw new \RuntimeException(sprintf(
-                'Cannot seek to offset % with whence %s',
+                'Cannot seek to offset %s with whence %s',
```

Comment:
Note that I fixed this as well.

Reply:
Backported to 1.x in https://github.com/guzzle/psr7/commit/868353eb117c202ee1f3b214eef9717e3d871297